dev-docs/a2-chef-server.md

```diff
@@ -0,0 +1,152 @@
+## Context
+The Automate deployment service can be used to deploy a standalone chef server without
+Automate. We consider this feature in its current state suitable for customer use as a
+services option. We have discussed the work to make deploying a single chef server via the
+Automate deployment service a full DIY product feature, but as of yet, we do not have
+concrete plans or schedule. Deploying HA chef server via the Automate deployment service
+is intended as a services option only.
+
+As of July 2019, the audience for the chef server deployed via the Automate deployment service consists
+of:
+
+1. opsworks, which deploys an all-in-one Automate with chef server on a single node
+2. the Customer Success team, which in July 2019 will start using the deployment service to deploy HA
+   chef server at selected customer sites,
+3. the Professional Services team, which is planning to start using the deployment service
+   to deploy HA chef server some time around August 2019.
+
+This document was written with the latter two groups in mind.
+
+## Deploy a new standalone chef server via Automate
+1. Install the chef-automate CLI tool. You will need one that contains commits: look for
+   hashes.
+2. Run `chef-automate init-config` to generate an editable config.toml.
+3. Add the following to the config.toml from 2:
+```
+[erchef.v1.sys.data_collector]
+enabled = false
+```
+Make any other config changes necessary.
+4. Run `chef-automate deploy --product chef-server config.toml` to install the chef
```

Comment:
[nit] Should we have them set the product in the toml instead of passing the product switch? Guess it doesn't really matter...

Reply:
config is better